### DISCUSSION TITLE
How to setup and which are the mandatory fields?

### DISCUSSION THREAD

**Discussion by Marcoletto** [gicamm/homeassistant-comelit#61] [Q&A] [answered]
Hi everyone,
I' ve the Comelit Alarm with vedo IP PCB (vedoIP installed today)
I don't have any HUB or other similar things.

Now, how can I setup the integration?

I know only the IP of vedo IP PCB
Vedo psw is the code that I used to arm o disarm the alarm?

All the other datas I don't know how to find (mqtt, ...HUB datas, ...)

Can you help me, please?

Thank you

```python



comelit:
  hub:
    host: HUB IP ADDRES #??
    port: 1883
    mqtt-user: hsrv-user #??
    mqtt-password: sf1nE9bjPc #??
    username: HUB USER #?? (e-mail that I use for the new Comelit user in APP?)
    password: HUB PASSWORD #?? (APP password?)
    serial: HUB SERIAL #??
    client: homeassistant
    scan_interval: 2
  vedo:
    host: VEDO IP ADDRESS #*192.168.178.xx*
    port: 80
    password: VEDO PASSWORD #?? (pws uset to arm & disarm the alarm by the alarm panel?)
    scan_interval: 30
    
    ```

**Comment by gicamm** [comment, marked as answer]
Hi Marco,
you can skip the configuration for the hub, so in your case it has to be something like

``` 
comelit:
  vedo:
    host: VEDO IP ADDRESS
    port: 80
    password: VEDO PASSWORD
    scan_interval: 30
    
``` 

The **password** is the numerical code that you use to arm and disarm it.
